opencog/cython/PythonEval.cc

```diff
@@ -579,50 +585,87 @@ void PythonEval::addSysPath(std::string path)
     //    PyRun_SimpleString(("sys.path += ['" + path + "']").c_str());
 }
 
+
+const int ABSOLUTE_IMPORTS_ONLY = 0;
+
 /**
 * Add all the .py files in the given directory as modules to __main__ and
 * keep the references in a dictionary (this->modules)
 */
-void PythonEval::add_module_directory(const boost::filesystem::path &p)
+void PythonEval::add_module_directory(const boost::filesystem::path &directory)
```

Comment:
As mentioned before, it does not make sense (to me) to have functions like this in PythonEval .. surely, some other class or function would be better? I'm thinking modularity -- don't make Eval do things that are not actually needed for evaluation.